pkg/vm/stackitem/stack_item.go

```diff
@@ -584,6 +543,11 @@ func (i *ArrayItem) Value() interface{} {
 	return i.value
 }
 
+// SetValue sets array item value
+func (i *ArrayItem) SetValue(items []StackItem) {
```

Comment:
I don't really like these setters for Array and Struct, we should either expose the `value` and let everyone use that or add some type-specific higher-level functions that would make getting/setting this not needed at all.

Reply:
We cannot expose `value` here because it is already declared in this package. Not sure it's worth renaming, so I added wrappers around `append` to `Clear`, `Append` and `Remove` elements in these types instead of `SetVelue`. Did you mean this way of solving?

Reply:
This new `Append` method is also useful in ceb068d5ddc7f8336702c1c875dc2bebf398b488